frontend/packages/console-shared/src/promql/cluster-dashboard.ts

```diff
@@ -44,7 +44,7 @@ const top25Queries = {
   [OverviewQuery.NODES_BY_MEMORY]:
     'topk(25, sort_desc(node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes))',
   [OverviewQuery.NODES_BY_STORAGE]:
-    'topk(25, sort_desc(sum(node_filesystem_size_bytes{fstype!="tmpfs",mountpoint!="/boot|/boot/efi"} - node_filesystem_avail_bytes{instance=~".*",fstype!="tmpfs",mountpoint!="/boot|/boot/efi"}) BY (instance)))',
+    'topk(25, sort_desc(sum(node_filesystem_size_bytes{fstype!="tmpfs|squashfs|nsfs",mountpoint!="/boot|/boot/efi|/usr|/var|/etc|/sysroot"} - node_filesystem_avail_bytes{instance=~".*",fstype!="tmpfs|squashfs|nsfs",mountpoint!="/boot|/boot/efi|/usr|/var|/etc|/sysroot"}) BY (instance)))',
```

Comment:
Is it intended to exclude `/boot|/boot/efi` only here and not in other queries?

Reply:
@paulfantom I don't have subject matter expertise on what mountpoints we need to exclude, so I just added on the ones the other had. Do you know who would be able to verify what's required?

Reply:
I'm not sure why we would exclude that in only one place.

@rawagner Do you happen to know the history of this query?

Reply:
I dont remember. I think at the beginning we had only `node_filesystem_size_bytes`, than the exclusions were added by @rebeccaalpert IIRC.

Reply:
Yup, Monitoring told me to add it initially, but now it's being questioned and I've been told they don't have subject matter expertise. I don't write these queries.